install/index.php

```diff
@@ -8,12 +8,11 @@
         
   <title>PHPBack Installation</title>
 
-    <link href="../public/bootstrap/css/bootstrap.css" rel="stylesheet">
-    <link href="../public/bootstrap/css/prettify.css" rel="stylesheet">
-    <link href="../public/css/flat-ui.css" rel="stylesheet">
+    <link href="../public/bootstrap/dist/css/bootstrap.css" rel="stylesheet">
```

Comment:
Not needed but it would be good to have all this `link` and `script` tags in single file and include it into places required. It would make it easy to maintain on the long run.